src/targetPacks/ILsrc/microsoft.netframework.referenceassemblies.net461/1.0.0-alpha-5/microsoft.netframework.referenceassemblies.net461.csproj

```diff
@@ -1,8 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <OutputPath>$(ArtifactsBinDir)microsoft.netframework.referenceassemblies.net461/1.0.0/</OutputPath>
-    <BaseIntermediateOutputPath>$(ArtifactsObjDir)microsoft.netframework.referenceassemblies.net461/1.0.0/</BaseIntermediateOutputPath>
+    <OutputPath>$(ArtifactsBinDir)microsoft.netframework.referenceassemblies.net461/1.0.0-alpha-5/</OutputPath>
```

Comment:
Same question here.  Why is this version changing?

Reply:
The version in previous updates was wrong. I realized it this time that its actually `alpha-5` which source-build-reference-packages references.

Reply:
Got it.  Thanks!